### ISSUE TITLE
1.8 RF

### ISSUE THREAD

**Issue by Vanhal** [MightyPirates/OpenComputers#953]
Hey, I also have a mod on 1.8 that uses RF as well.
I had a user mention that they tried to use one of my generators to try power an open computer and it didn't work (obviously),
It might be an idea to work together and get a common version and get a bit of an RF lib going for 1.8. (Since they really don't seem like they want to port to 1.8)

The version that I'm currently using is here: https://github.com/Vanhal/RedstoneFlux-API-1.8
Let me know.

**Comment by fnuecke** [comment]
I enabled injection of the `IEnergyReceiver` interface into power receiving blocks in the 1.8 branch ([devbuilds here](http://ci.cil.li/job/OpenComputers-dev-MC1.8/)). Will require the `CoFHAPI|energy` API to be present via some other mod (OC doesn't ship any third-party APIs). From what I saw the only change was `ForgeDirection` -> `EnumFacing`, so the implementation should be compatible and work. Let me know if it works, or how to best test it, thanks!
